lib/Doctrine/ORM/EntityRepository.php

```diff
@@ -162,13 +162,6 @@ public function createNativeNamedQuery($queryName)
      */
     public function clear()
     {
-        Deprecation::trigger(
-            'doctrine/orm',
-            'https://github.com/doctrine/orm/issues/8460',
-            'Calling %s() is deprecated and will not be supported in Doctrine ORM 3.0.',
-            __METHOD__
-        );
-
         if (! class_exists(PersistentObject::class)) {
             throw NotSupported::createForPersistence3(sprintf(
                 'Partial clearing of entities for class %s',
```

Comment:
I'm not sure what should be done here… should we create persistence 3.3 with this PR reverted: https://github.com/doctrine/persistence/pull/174 ?

Reply:
> I'm not sure what should be done here… should we create persistence 3.3 with this PR reverted: https://github.com/doctrine/persistence/pull/174 ?

That would be a BC break.

I think, it's fine to not reintroduce that into Persistence. At this point, partial clearing is a feature the ORM adds on top of the Persistence interfaces.